pages/DSNP/Operations.md

```diff
@@ -0,0 +1,36 @@
+# Operations
+
+DSNP implementations perform well-defined DSNP Operations and generate DSNP State Change Records.
+
+## Control Keys and Proofs
+
+Each invocation of a DSNP Operation MUST have verifiable approval of the acting principal(s) via a Control Key Ownership Proof.
+The precise data type and representation of both the Control Key and the Control Key Ownership Proof MUST be defined by each DSNP implementation.
+For example, an implementation might use the public key of an asymmetric key pair as a control key, and provide a proof for each operation by producing a cryptographic signature of the user's DSNP Identifier and some nonce value.
+
+Where operations are listed as using control keys or ownership proofs as input parameters, this indicates that they should be provided in addition to those providing invocation authentication.
```

Comment:
It's not clear to me what "they" refers to in this sentence.

Reply:
Will fix.